### PR TITLE
Revamp "AnimationContext" Docs, now called "AnimTaskQueue"

### PR DESCRIPTION
**Pull request by engineer124**
This updates the Animation docs to be in sync with OoT. The only difference between these two systems is in `AnimTask_ActorMove`, where the OoT parameter: `diffScaleY` now scales all 3 directions: xyz in MM, not just y. So that was renamed to just `diffScale`.

See this PR in OoT: https://github.com/zeldaret/oot/pull/1941

### CODE REVIEW

**Comment by fig02** [comment]
Just an fyi I never finished my documentation on this in oot, and intend to get back to it shortly (after the Z Targeting stuff comes to an end)

My plan is to make the terminology around "Actor Move" be more consistent, and possibly rename it all to "ActorMovement". Currently some comments in oot refer to it as "anim translation", "actor move", "move actor" etc so I wanted to clean that up, and also rename `ANIM_FLAG_PLAYER_SETMOVE`.

Not saying you have to wait for that to do these docs here, but just wanted to let you know.

File: include/z64animation.h [hunk 1]
```diff
@@ -37,7 +37,7 @@ typedef enum AnimationMode {
     /* 5 */ ANIMMODE_LOOP_PARTIAL_INTERP
 } AnimationMode;
 
-typedef enum {
+typedef enum AnimationTapers {
```
Comment (hensldm) [inline include/z64animation.h:40]:
```suggestion
typedef enum AnimationTaper {
```
Could you also update the name at the bottom of the enum to be singular as well?

**Review by hensldm — COMMENTED** [review]
1 tiny nit

**Comment by engineer124** [comment]
> Just an fyi I never finished my documentation on this in oot, and intend to get back to it shortly (after the Z Targeting stuff comes to an end)

No problems, we can always update the docs further at a later point

**Review by Dragorn421 — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Review by hensldm — APPROVED** [review]

**Pull request merged**